talentlink-connector-product/README.md

```diff
@@ -0,0 +1,37 @@
+# TalentLink Connector
```

Comment:
@andreasbalsiger do you have any non-technical feedback for this new connector description?